reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+
+### Add a member
+
+After [creating a networkgroup](#create-a-networkgroup), you can add members to it. A member is an app, an add-on or an [external node category](TODO: Redirect to doc). You can set `--type` to `'application'`, `'addon'` or `'external'` according to your needs.
+
+For example, to add an app to the networkgroup, you can do:
+
+```sh
+clever ng --ng 'first-ng' members add --member-id 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' --type 'application' --domain-name 'my-app-1' --label 'My app 1'
+```
+
+```log
+Successfully added member 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- `--label` is optional, but it's a good practice to add it.
+
+### List members in a networkgroup
+
+To list the members in a networkgroup, run `clever ng members list`. For example, after ["Add a member"](#add-a-member), you could run:
+
+```sh
+clever ng --ng 'first-ng' members list
+```
+
+You should get something like:
+
+```log
+Member ID                                 Member Type                Label                                     Domain Name
+───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+app_d60ef23d-80d7-4095-b8d7-7f42bea137ea  application                My app 1                                  my-app-1
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- [`--json` or `-j` shows output as JSON](#--json-or--j-shows-output-as-json)
+
+### Add an external node category
+
+External node categories are <TODO>. You can have plenty of them, as it helps keeping this organized.
+
+Adding an external node category is very similar to the example in ["Add a member"](#add-a-member), except you'll use a custom member ID and `'external'` for the `--type`.
+
+Here is an example:
+
+```sh
+clever ng --ng 'first-ng' members add --member-id 'dev-laptops-category' --type 'external' --domain-name 'dev-laptops' --label 'Developer laptops'
+```
+
+> I voluntarily used `--member-id` and `--domain-name` but you can have similar ones if you want.
+
+```log
+Successfully added member 'dev-laptops-category' to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tip:** [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+
+### Join a networkgroup
+
+To join a networkgroup, you need to [add an external node category](#add-an-external-node-category) and run `sudo clever ng join`. This command uses `wg-quick` under the hood, which needs privileges to create network interfaces. That's why you need to use `sudo`.
+
+> Clever Cloud's networkgroups use WireGuard®. Therefore, this command requires WireGuard® commands available on your computer.
+>
+> Follow instructions at <https://www.wireguard.com/install/> to install it.
```

Comment:
Install WireGuard by following the instructions at <https://www.wireguard.com/install/>

(okay this is nitpicking at this point)